app/package.json

```diff
@@ -20,6 +20,7 @@
     "react-fontawesome": "^1.7.1",
     "react-router-dom": "^6.21.1",
     "react-scripts": "5.0.1",
+    "react-uuid": "^2.0.0",
```

Comment:
Király vagy, hogy az uuid magoldást választottad a random szám helyett! Viszont, ez a package már deprecated, szóval ezt ne használd. A weekly downloads is elég alacsony, mutatja, hogy ez nem egy megbízható package.

<img width="1192" alt="Screenshot 2024-01-16 at 16 19 29" src="https://github.com/orsolyahrubo/scrabble/assets/52178754/28f8a413-d734-42c3-a34e-73fbe9cc160e">

Helyette használd a `uuid` package-t légyszi, ez sokkal széleskörűbben használt.

<img width="1192" alt="Screenshot 2024-01-16 at 16 20 30" src="https://github.com/orsolyahrubo/scrabble/assets/52178754/39d1b7e1-0a64-42e2-aa9c-2303259fbba2">

Itt a v1 functiont használd